repos/spack_repo/builtin/packages/libmesh/package.py

```diff
@@ -156,7 +158,7 @@ class Libmesh(AutotoolsPackage):
     depends_on("slepc", when="+slepc")
     depends_on("petsc", when="+petsc")
     depends_on("tbb", when="threads=tbb")
-    depends_on("vtk", when="+vtk")
+    depends_on("vtk+mpi", when="+vtk")
```

Comment:
```suggestion
    depends_on("vtk+mpi", when="+vtk+mpi")
```
@rbberger, I think this is your suggestion? I'll wait for @capitalaslash to respond or merge this incremental change.

Reply:
see https://github.com/spack/spack-packages/pull/779#discussion_r2293757087